### DISCUSSION TITLE
Installation Post 18.02

### DISCUSSION THREAD

**Discussion by mfenton01** [opendcim/openDCIM#1372] [Q&A] [answered]
My installation when pointing to the new installation  version just redirects me from my test to my production .  Which i do not understand because the configuration has no knowledge of that host .   I point it back to the previous version 18.02 ..  it works as it should .  ssl Vhost is read hits basic authentication then loads DCIM .  Then when I point to the anything newer well it redirects me and then tells me cant find the lnstall.php . Any Ideas ?

**Comment by samilliken** [comment, marked as answer]
If you copied the db from production, the InstallURL would be set the production's.   You'd have to manually poke that to fix it.   From MySQL command line client:

```
update fac_Config set Value='https://my-dev-url' where Parameter='InstallURL';
```
